lib/platform/repositories/movie_repository.dart

```diff
@@ -26,4 +26,7 @@ class MovieRepository {
 
   Future<Credits> getMovieCredits(int id) async =>
       await api.getMovieCredits(id);
+
+  Future<List<Movie>> getMoviesByGenre(int genreId, {int page = 1}) async =>
+      await api.getMoviesByGenre(genreId, page: page);
```

Comment:
```suggestion
  Future<List<Movie>> getMoviesByGenre(int genreId, {int page = 1}) => api.getMoviesByGenre(genreId, page: page);
```
có thể viết ngắn gọn lại như này ạ